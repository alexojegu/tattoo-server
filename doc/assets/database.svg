<svg
  xmlns="http://www.w3.org/2000/svg"
  width="770px"
  height="362px"
  viewBox="0 0 770 362"
  xmlns:xlink="http://www.w3.org/1999/xlink"
  style="overflow: hidden; display: block; width: 770px; height: 362px"
>
  <defs>
    <path
      d="M 16,-8 L 16,8 M 6,-8 L 6,8 Z"
      stroke="rgb(51,102,153)"
      stroke-opacity="1"
      stroke-width="1"
      stroke-linecap="butt"
      stroke-linejoin="miter"
      stroke-miterlimit="10"
      id="ygc95_0"
    />
    <g id="ygc95_1">
      <circle
        fill="#fff"
        stroke="rgb(51,102,153)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
        r="5"
        cx="15"
        cy="0"
      />
      <path
        d="M 6,-8 L 6,8"
        stroke="rgb(51,102,153)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
    </g>
    <path
      d="M 15,8 L 15,-8 M 10,0 L -2,8 M 10,0 L -2,0 M 10,0 L -2,-8"
      stroke="rgb(51,102,153)"
      stroke-opacity="1"
      stroke-width="1"
      stroke-linecap="butt"
      stroke-linejoin="miter"
      stroke-miterlimit="10"
      id="ygc95_2"
    />
    <g id="ygc95_3">
      <circle
        fill="#fff"
        stroke="rgb(51,102,153)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
        r="5"
        cx="15"
        cy="0"
      />
      <path
        d="M 6,-8 L 6,8"
        stroke="rgb(51,102,153)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
    </g>
    <g id="ygc95_4">
      <circle
        fill="#fff"
        stroke="rgb(51,102,153)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
        r="5"
        cx="15"
        cy="0"
      />
      <path
        d="M 6,-8 L 6,8"
        stroke="rgb(51,102,153)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
    </g>
    <path
      d="M 15,8 L 15,-8 M 10,0 L -2,8 M 10,0 L -2,0 M 10,0 L -2,-8"
      stroke="rgb(51,102,153)"
      stroke-opacity="1"
      stroke-width="1"
      stroke-linecap="butt"
      stroke-linejoin="miter"
      stroke-miterlimit="10"
      id="ygc95_5"
    />
    <path
      d="M 15,8 L 15,-8 M 10,0 L -2,8 M 10,0 L -2,0 M 10,0 L -2,-8"
      stroke="rgb(51,102,153)"
      stroke-opacity="1"
      stroke-width="1"
      stroke-linecap="butt"
      stroke-linejoin="miter"
      stroke-miterlimit="10"
      id="ygc95_6"
    />
    <g id="ygc95_7">
      <circle
        fill="#fff"
        stroke="rgb(51,102,153)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
        r="5"
        cx="15"
        cy="0"
      />
      <path
        d="M 6,-8 L 6,8"
        stroke="rgb(51,102,153)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
    </g>
  </defs>
  <g
    style="pointer-events: visiblePainted"
    transform="translate(25 347.25)"
    image-rendering="auto"
    shape-rendering="auto"
  >
    <g>
      <path
        fill="none"
        stroke="rgb(51,102,153)"
        d="M 150,-250 L 190,-250"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
      <use xlink:href="#ygc95_0" transform="matrix(-0.75 0 0 -0.75 190 -250)" />
      <use xlink:href="#ygc95_1" transform="matrix(0.75 0 0 0.75 150 -250)" />
    </g>
    <g>
      <path
        fill="none"
        stroke="rgb(51,102,153)"
        d="M 340,-250 L 570,-250"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
      <use xlink:href="#ygc95_2" transform="matrix(-0.75 0 0 -0.75 570 -250)" />
      <use xlink:href="#ygc95_3" transform="matrix(0.75 0 0 0.75 340 -250)" />
    </g>
    <g>
      <path
        fill="none"
        stroke="rgb(51,102,153)"
        d="M 265,-185 L 265,-122.5 L 380,-122.5"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
      <use xlink:href="#ygc95_4" transform="matrix(-0.75 0 0 -0.75 380 -122.5)" />
      <use xlink:href="#ygc95_5" transform="matrix(0 0.75 -0.75 0 265 -185)" />
    </g>
    <g>
      <path
        fill="none"
        stroke="rgb(51,102,153)"
        d="M 530,-122.5 L 645,-122.5 L 645,-185"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
      <use xlink:href="#ygc95_6" transform="matrix(0 0.75 -0.75 0 645 -185)" />
      <use xlink:href="#ygc95_7" transform="matrix(0.75 0 0 0.75 530 -122.5)" />
    </g>
    <g transform="translate(0 -322.5)">
      <path d="M 0,10 Q 0,0 10,0 L 140,0 Q 150,0 150,10 L 150,24 L 0,24 Z" fill="rgb(183,201,227)" fill-opacity="1" />
      <path
        d="M 150,24 L 150,135 Q 150,145 140,145 L 10,145 Q 0,145 0,135 L 0,24 Z"
        fill="rgb(232,238,247)"
        fill-opacity="1"
      />
      <path
        d="M 0,10 Q 0,0 10,0 L 140,0 Q 150,0 150,10 L 150,135 Q 150,145 140,145 L 10,145 Q 0,145 0,135 Z M 0,24 L 150,24"
        fill="none"
        stroke="rgb(0,0,0)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
      <g transform="translate(0 5)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="middle"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(75 0)"
        >
          <tspan dy="0.9166666666666666em" x="0">account</tspan>
        </text>
      </g>
      <g transform="translate(5 26)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">id</tspan>
        </text>
      </g>
      <g transform="translate(5 42)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">email</tspan>
        </text>
      </g>
      <g transform="translate(5 58)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">password</tspan>
        </text>
      </g>
      <g transform="translate(5 74)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">name</tspan>
        </text>
      </g>
      <g transform="translate(5 90)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">image</tspan>
        </text>
      </g>
      <g transform="translate(5 106)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">created</tspan>
        </text>
      </g>
      <g transform="translate(5 122)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">updated</tspan>
        </text>
      </g>
    </g>
    <g transform="translate(190 -315)">
      <path d="M 0,10 Q 0,0 10,0 L 140,0 Q 150,0 150,10 L 150,24 L 0,24 Z" fill="rgb(183,201,227)" fill-opacity="1" />
      <path
        d="M 150,24 L 150,120 Q 150,130 140,130 L 10,130 Q 0,130 0,120 L 0,24 Z"
        fill="rgb(232,238,247)"
        fill-opacity="1"
      />
      <path
        d="M 0,10 Q 0,0 10,0 L 140,0 Q 150,0 150,10 L 150,120 Q 150,130 140,130 L 10,130 Q 0,130 0,120 Z M 0,24 L 150,24"
        fill="none"
        stroke="rgb(0,0,0)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
      <g transform="translate(0 5)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="middle"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(75 0)"
        >
          <tspan dy="0.9166666666666666em" x="0">artist</tspan>
        </text>
      </g>
      <g transform="translate(5 26)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">id</tspan>
        </text>
      </g>
      <g transform="translate(5 42)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">website</tspan>
        </text>
      </g>
      <g transform="translate(5 58)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">instagram</tspan>
        </text>
      </g>
      <g transform="translate(5 74)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">facebook</tspan>
        </text>
      </g>
      <g transform="translate(5 90)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">created</tspan>
        </text>
      </g>
      <g transform="translate(5 106)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">updated</tspan>
        </text>
      </g>
    </g>
    <g transform="translate(380 -235)">
      <path d="M 0,10 Q 0,0 10,0 L 140,0 Q 150,0 150,10 L 150,24 L 0,24 Z" fill="rgb(183,201,227)" fill-opacity="1" />
      <path
        d="M 150,24 L 150,215 Q 150,225 140,225 L 10,225 Q 0,225 0,215 L 0,24 Z"
        fill="rgb(232,238,247)"
        fill-opacity="1"
      />
      <path
        d="M 0,10 Q 0,0 10,0 L 140,0 Q 150,0 150,10 L 150,215 Q 150,225 140,225 L 10,225 Q 0,225 0,215 Z M 0,24 L 150,24"
        fill="none"
        stroke="rgb(0,0,0)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
      <g transform="translate(0 5)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="middle"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(75 0)"
        >
          <tspan dy="0.9166666666666666em" x="0">studio</tspan>
        </text>
      </g>
      <g transform="translate(5 26)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">id</tspan>
        </text>
      </g>
      <g transform="translate(5 42)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">name</tspan>
        </text>
      </g>
      <g transform="translate(5 58)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">image</tspan>
        </text>
      </g>
      <g transform="translate(5 74)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">website</tspan>
        </text>
      </g>
      <g transform="translate(5 90)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">instagram</tspan>
        </text>
      </g>
      <g transform="translate(5 106)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">facebook</tspan>
        </text>
      </g>
      <g transform="translate(5 122)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">phone</tspan>
        </text>
      </g>
      <g transform="translate(5 138)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">address</tspan>
        </text>
      </g>
      <g transform="translate(5 154)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">location</tspan>
        </text>
      </g>
      <g transform="translate(5 170)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">about</tspan>
        </text>
      </g>
      <g transform="translate(5 186)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">created</tspan>
        </text>
      </g>
      <g transform="translate(5 202)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">updated</tspan>
        </text>
      </g>
    </g>
    <g transform="translate(570 -315)">
      <path d="M 0,10 Q 0,0 10,0 L 140,0 Q 150,0 150,10 L 150,24 L 0,24 Z" fill="rgb(183,201,227)" fill-opacity="1" />
      <path
        d="M 150,24 L 150,120 Q 150,130 140,130 L 10,130 Q 0,130 0,120 L 0,24 Z"
        fill="rgb(232,238,247)"
        fill-opacity="1"
      />
      <path
        d="M 0,10 Q 0,0 10,0 L 140,0 Q 150,0 150,10 L 150,120 Q 150,130 140,130 L 10,130 Q 0,130 0,120 Z M 0,24 L 150,24"
        fill="none"
        stroke="rgb(0,0,0)"
        stroke-opacity="1"
        stroke-width="1"
        stroke-linecap="butt"
        stroke-linejoin="miter"
        stroke-miterlimit="10"
      />
      <g transform="translate(0 5)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="middle"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(75 0)"
        >
          <tspan dy="0.9166666666666666em" x="0">tattoo</tspan>
        </text>
      </g>
      <g transform="translate(5 26)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">id</tspan>
        </text>
      </g>
      <g transform="translate(5 42)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">image</tspan>
        </text>
      </g>
      <g transform="translate(5 58)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">width</tspan>
        </text>
      </g>
      <g transform="translate(5 74)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">height</tspan>
        </text>
      </g>
      <g transform="translate(5 90)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">created</tspan>
        </text>
      </g>
      <g transform="translate(5 106)">
        <text
          font-family="'Arial'"
          font-size="12px"
          font-style="normal"
          font-weight="normal"
          text-decoration="none"
          text-anchor="start"
          fill="rgb(0,0,0)"
          fill-opacity="1"
          transform="translate(0 1)"
        >
          <tspan dy="0.9166666666666666em" x="0">updated</tspan>
        </text>
      </g>
    </g>
  </g>
</svg>
