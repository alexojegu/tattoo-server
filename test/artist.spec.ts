import { ARTISTS } from "./__requests__/artist";
import { seed, wipe } from "./__utils__/database";
import { post } from "./__utils__/request";

describe("GraphQL API artista", () => {
    describe("Consultas de artista", () => {
        beforeAll(async () => {
            await seed();
        });

        afterAll(async () => {
            await wipe();
        });

        test("Obtiene los primeros 5 artistas", async () => {
            const variables = { limit: 5, tattooLimit: 3 };
            const response = await post(ARTISTS, variables);

            expect(await response.json()).toMatchSnapshot();
        });

        test("Obtiene los ultimos 5 artistas", async () => {
            const variables = { limit: 5, cursor: "eyJpZCI6Nn0=", tattooLimit: 3 };
            const response = await post(ARTISTS, variables);

            expect(await response.json()).toMatchSnapshot();
        });
    });
});
