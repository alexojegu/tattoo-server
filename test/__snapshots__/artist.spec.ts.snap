// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GraphQL API artista Consultas de artista Obtiene un artista por id 1`] = `
Object {
  "data": Object {
    "artist": Object {
      "account": Object {
        "id": "1",
      },
      "created": "2020-02-01T12:30:30.000Z",
      "facebook": "https://aol.com",
      "id": "1",
      "instagram": "http://tumblr.com",
      "studio": Object {
        "id": "1",
      },
      "tattoos": Object {
        "nodes": Array [
          Object {
            "id": "1",
          },
        ],
      },
      "updated": "2020-03-01T12:30:30.000Z",
      "website": "http://china.com.cn",
    },
  },
}
`;

exports[`GraphQL API artista Consultas de artista Obtiene los primeros 5 artistas 1`] = `
Object {
  "data": Object {
    "artists": Object {
      "nodes": Array [
        Object {
          "account": Object {
            "id": "59",
          },
          "created": "2020-03-01T12:30:30.000Z",
          "facebook": "https://moonfruit.com",
          "id": "30",
          "instagram": null,
          "studio": Object {
            "id": "30",
          },
          "tattoos": Object {
            "nodes": Array [],
          },
          "updated": null,
          "website": null,
        },
        Object {
          "account": Object {
            "id": "57",
          },
          "created": "2020-02-29T12:30:30.000Z",
          "facebook": "http://uiuc.edu",
          "id": "29",
          "instagram": "http://nasa.gov",
          "studio": Object {
            "id": "29",
          },
          "tattoos": Object {
            "nodes": Array [
              Object {
                "id": "45",
              },
              Object {
                "id": "44",
              },
              Object {
                "id": "43",
              },
            ],
          },
          "updated": "2020-03-15T12:30:30.000Z",
          "website": "http://latimes.com",
        },
        Object {
          "account": Object {
            "id": "55",
          },
          "created": "2020-02-28T12:30:30.000Z",
          "facebook": "https://uol.com.br",
          "id": "28",
          "instagram": null,
          "studio": Object {
            "id": "27",
          },
          "tattoos": Object {
            "nodes": Array [],
          },
          "updated": null,
          "website": "https://slate.com",
        },
        Object {
          "account": Object {
            "id": "53",
          },
          "created": "2020-02-27T12:30:30.000Z",
          "facebook": "https://google.it",
          "id": "27",
          "instagram": "http://yahoo.co.jp",
          "studio": Object {
            "id": "27",
          },
          "tattoos": Object {
            "nodes": Array [
              Object {
                "id": "40",
              },
              Object {
                "id": "39",
              },
              Object {
                "id": "38",
              },
            ],
          },
          "updated": "2020-03-14T12:30:30.000Z",
          "website": null,
        },
        Object {
          "account": Object {
            "id": "51",
          },
          "created": "2020-02-26T12:30:30.000Z",
          "facebook": "http://harvard.edu",
          "id": "26",
          "instagram": null,
          "studio": Object {
            "id": "26",
          },
          "tattoos": Object {
            "nodes": Array [],
          },
          "updated": null,
          "website": "http://omniture.com",
        },
      ],
      "page": Object {
        "cursor": "eyJpZCI6MjZ9",
        "next": true,
      },
    },
  },
}
`;

exports[`GraphQL API artista Consultas de artista Obtiene los ultimos 5 artistas 1`] = `
Object {
  "data": Object {
    "artists": Object {
      "nodes": Array [
        Object {
          "account": Object {
            "id": "9",
          },
          "created": "2020-02-05T12:30:30.000Z",
          "facebook": "http://msu.edu",
          "id": "5",
          "instagram": "http://wufoo.com",
          "studio": Object {
            "id": "5",
          },
          "tattoos": Object {
            "nodes": Array [
              Object {
                "id": "6",
              },
              Object {
                "id": "5",
              },
              Object {
                "id": "4",
              },
            ],
          },
          "updated": "2020-03-03T12:30:30.000Z",
          "website": "https://forbes.com",
        },
        Object {
          "account": Object {
            "id": "7",
          },
          "created": "2020-02-04T12:30:30.000Z",
          "facebook": "http://rambler.ru",
          "id": "4",
          "instagram": null,
          "studio": Object {
            "id": "3",
          },
          "tattoos": Object {
            "nodes": Array [],
          },
          "updated": null,
          "website": "https://blogger.com",
        },
        Object {
          "account": Object {
            "id": "5",
          },
          "created": "2020-02-03T12:30:30.000Z",
          "facebook": "http://state.tx.us",
          "id": "3",
          "instagram": "https://imgur.com",
          "studio": Object {
            "id": "3",
          },
          "tattoos": Object {
            "nodes": Array [
              Object {
                "id": "3",
              },
              Object {
                "id": "2",
              },
            ],
          },
          "updated": "2020-03-02T12:30:30.000Z",
          "website": null,
        },
        Object {
          "account": Object {
            "id": "3",
          },
          "created": "2020-02-02T12:30:30.000Z",
          "facebook": "http://friendfeed.com",
          "id": "2",
          "instagram": null,
          "studio": Object {
            "id": "2",
          },
          "tattoos": Object {
            "nodes": Array [],
          },
          "updated": null,
          "website": "https://ameblo.jp",
        },
        Object {
          "account": Object {
            "id": "1",
          },
          "created": "2020-02-01T12:30:30.000Z",
          "facebook": "https://aol.com",
          "id": "1",
          "instagram": "http://tumblr.com",
          "studio": Object {
            "id": "1",
          },
          "tattoos": Object {
            "nodes": Array [
              Object {
                "id": "1",
              },
            ],
          },
          "updated": "2020-03-01T12:30:30.000Z",
          "website": "http://china.com.cn",
        },
      ],
      "page": Object {
        "cursor": "eyJpZCI6MX0=",
        "next": false,
      },
    },
  },
}
`;
